documentation/modules/overview/con-configuration-points-connect.adoc

```diff
@@ -5,127 +5,257 @@
 [id="configuration-points-connect_{context}"]
 = Kafka Connect configuration
 
-A basic Kafka Connect configuration requires a bootstrap address to connect to a Kafka cluster, and encryption and authentication details.
+[role="_abstract"]
+You can use the `KafkaConnect` custom resource to configure a Kafka Connect instance.
+Using this resource, you can configure the worker nodes that comprise a Kafka Connect cluster and build a container image with connector plugin configuration.
+You can also enable KafkaConnectors in your `KafkaConnect` configuration to be able to use the `KafkaConnector` resource to manage connector plugins.
```

Comment:
To enable KafkaConnectors, use the `strimzi.io/use-connector-resources: "true"` annotation.

Reply:
@laidan6000 - this is just the abstract of the section. The enabling bit is described in a sub-section. 

Changed to:

> You can also enable KafkaConnectors in your `KafkaConnect` configuration. KafkaConnectors are an alternative to managing connectors using the Kafka Connect REST API.